src/components/FeaturePanel/Climbing/contexts/osmToClimbingRoutes.ts

```diff
@@ -55,6 +55,7 @@ export const osmToClimbingRoutes = (feature: Feature): Array<ClimbingRoute> => {
       difficulty: getDifficulty(route.tags),
       paths,
       photoToKeyMap,
+      author: route.tags.author,
```

Comment:
Can you use `feature.tags.author` instead of adding a new field? We will have to move all of it nevertheless.

Reply:
I wanted but it needs broader refactoring. Right now we have temp object for editing with ClimbingRoute type, so this whole needs to be refactored.